<launch>
<node pkg="beginner_tutorials" type="my_service"  name="part_detection" args="true" output="screen"/>
</launch>
